update studio 3.6.0 (#1902)
vesoft-inc/nebula-docs: docs-2.0/nebula-studio/deploy-connect/st-ug-connect.md

- The NebulaGraph services and Studio are started. For more information, see [Deploy Studio](st-ug-deploy.md).

- You have the local IP address and the port used by the Graph service of NebulaGraph. The default port is `9669`.  

- You have a NebulaGraph account and its password.

## Procedure

To connect Studio to NebulaGraph, follow these steps:

1. Type `http://<ip_address>:7001` in the address bar of your browser.

  The following login page shows that Studio is successfully connected to NebulaGraph.

  ![NebulaGraph Studio Login page](https://docs-cdn.nebula-graph.com.cn/figures/studio_login_230130_en.png)

2. On the **Config Server** page of Studio, configure these fields:

  - **Graphd IP address**: Enter the IP address of the Graph service of NebulaGraph. For example, `192.168.10.100`.

    !!! note

>         - When NebulaGraph and Studio are deployed on the same machine, you must enter the IP address of the machine, but not `127.0.0.1` or `localhost`.

Reviewer: ```suggestion
        - When NebulaGraph and Studio are deployed on the same machine, you must enter the IP address of the machine, instead of `127.0.0.1` or `localhost`.
```

>         - When connecting a NebulaGraph database on a new tab, The new session will overwrite the sessions of the old TAB. If you need to log in to multiple NebulaGraph databases at the same time, you can use different browsers or non-trace mode.

Reviewer: ```suggestion
        - When connecting to a NebulaGraph database on a new tab, a new session will overwrite the sessions of the old TAB. If you need to log in to multiple NebulaGraph databases simultaneously, you can use a different browser or non-trace mode.
```
